lib/utils.pm

```diff
@@ -1414,6 +1414,13 @@ sub reconnect_mgmt_console {
             send_key 'ret';
         }
     }
+    elsif (check_var('ARCH', 'aarch64')) {
```

Comment:
What about `is_aarch64` ?

Reply:
I wanted to keep it the same as we have above for other arches.